src/utils/event-utils.ts

```diff
@@ -31,6 +31,25 @@ export const CAMPAIGN_PARAMS = [
     'rdt_cid', // reddit
 ]
 
+export const EVENT_TO_PERSON_PROPERTIES = [
+    // mobile params
+    '$app_build',
+    '$app_name',
+    '$app_namespace',
+    '$app_version',
```

Comment:
There are many more params than that such as `$device_manufacturer`, `$device_model`, etc, I wonder if this list is exhaustive to only what this SDK specifically collects?

Reply:
Those other ones aren't converted into person properties though, or are they? This list comes straight from plugin-server.

Reply:
I didn't check so the plugin-server should be up to date and all good.